src/kernel/src/fs/file.rs

```diff
@@ -8,13 +8,15 @@ use std::sync::Arc;
 
 /// An implementation of `file` structure.
 #[derive(Debug)]
+#[allow(dead_code)]
 pub struct VFile {
     ty: VFileType,                    // f_type
     data: Arc<dyn Any + Send + Sync>, // f_data
     ops: &'static VFileOps,           // f_ops
     flags: VFileFlags,                // f_flag
 }
 
+#[allow(dead_code)]
```

Comment:
This should not apply on the implementation because it can hide the things that we don't actually need.

Reply:
Seems like to missed this one.